Files/Views/ModernShellPage.xaml.cs

```diff
@@ -891,6 +891,16 @@ private async void KeyboardAccelerator_Invoked(KeyboardAccelerator sender, Keybo
                     await storageHistoryHelpers.TryRedo();
                     break;
 
+                case (true, true, false, true, VirtualKey.T): // ctrl + shif + t, restore recently closed tab
```

Comment:
Here

Reply:
What does `tabInstance` represent?

Reply:
Tab instance is true whether `CurrentPageType` is `GenericFileBrowser` or `GenericFileBrowser`

Reply:
Ok, I solved it by replacing the bool with discard `_` symbol